packages/mattermost-redux/src/types/apps.ts

```diff
@@ -32,6 +32,7 @@ export type AppManifest = {
     app_type: AppType;
     version?: string;
     homepage_url?: string;
+    icon?: string;
```

Comment:
Is this being used? (and if we do - why not use it directly instead of defining similar fields on `MarketplacePlugin`/MarketplaceApp` ?

Reply:
No, it's not used right now. I added it for consistency with the server code.

`icon_url` from  `MarketplaceApp` is the one being used, as the server pre fills it with a sanitized value.